cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1204,8 +1204,11 @@ def default_for(
                    The workflow polls for task status (no task messaging)
                 ssh
                    Use non-interactive ssh for task communications
+                https
+                   Via the running Hub proxy and/or UI-Server (requires UI
+                   Server installation)
```

Comment:
Can put an added label just on the `https` method:

```suggestion
                   Server installation)
                   
                   .. versionadded:: 8.2.0
```